qiskit_experiments/base_analysis.py

```diff
@@ -25,6 +27,36 @@ class BaseAnalysis(ABC):
     # Expected experiment data container for analysis
     __experiment_data__ = ExperimentData
 
+    def __init__(self, **options):
+        """Initialize a base analysis class
+
+        Args:
+            options: kwarg options for analysis.
```

Comment:
I agree with having more description would be great, but it seems difficult. The `Options` class defines nothing.
https://github.com/Qiskit/qiskit-terra/blob/9fd5436ba30b98d3bfd2578cc7f4ce564dfffd55/qiskit/providers/options.py#L18-L27

This options is provider dependent, but the experiment is provider agnostic (since this is analysis option, this can be experiment dependent).